ReleaseVersion.xml

```diff
@@ -5,6 +5,6 @@
     - divergent versions, but for the moment this keeps things simpler.
     -->
   <PropertyGroup>
-    <Version>3.3.0-alpha01</Version>
+    <Version>3.3.0-alpha02</Version>
```

Comment:
Just double checking that this should be here. I think it should because now we'll have only one branch, but to make sure.

Reply:
Yup, that's correct. It's about to be changed to 3.3.0 though :)